utils/responses.js

```diff
@@ -0,0 +1,19 @@
+exports.success = (req, res, message, status) => {
+  let statusCode = status || 200;
+  let statusMessage = message || '';
+  res.status(statusCode).send({
+    error: false,
+    status: statusCode,
+    data: statusMessage,
+  });
+};
+
+exports.error = (req, res, message, status) => {
+  let statusCode = status || 200;
+  let statusMessage = message || '';
+  res.status(statusCode).send({
```

Comment:
I think that instead of using the method `.send` it'd be better if we use `.json`, because with the send method you have to set the `Content-Type` header, and the JSON method automatically sets the header correctly to send JSON. I noticed you're trying to send a JSON.

Reply:
It's ok to me, the JSON way works correctly.